lib/ts/recipe/session/sessionAuth.tsx

```diff
@@ -34,6 +34,10 @@ import type { PropsWithChildren } from "react";
 import type { ClaimValidationError } from "supertokens-web-js/recipe/session";
 
 export type SessionAuthProps = {
+    /**
+     * Initial context that is rendered on a server side (SSR).
+     */
+    initialSessionAuthContext?: SessionContextType;
```

Comment:
since we want to always want to mark this as an SSR context and since there'll be no real claim validation info in it, I'd prefer if those were not part of the input. This should use a separate type

Reply:
I'd prefer to keep it the same time as we can't guarantee that `initialSessionAuthContext` is coming from `getInitialSessionAuthContext`, in some case the user might implement a custom logic that will build an `initialSessionAuthContext` with `invalidClaims` for example